src/pkg/packager/prepare.go

```diff
@@ -37,7 +38,8 @@ import (
 	"github.com/zarf-dev/zarf/src/types"
 )
 
-var imageCheck = regexp.MustCompile(`(?mi)"image":"([^"]+)"`)
+var imageCheck = regexp.MustCompile(`(?mi)"image":"((([a-z0-9._-]+)/)?([a-z0-9._-]+)(:([a-z0-9._-]+))?)"`)
+var imagePodCheck = regexp.MustCompile(`(?mi)^(([a-z0-9._-]+)/)?([a-z0-9._-]+)(:([a-z0-9._-]+))?$`)
```

Comment:
I believe this regex doesn't account for registry domain names or potentially a local docker registry (ip:port). While I am not a regex expert - I believe something to the effect of the suggestion below to accomplish the intent. 

```suggestion
var imagePodCheck = regexp.MustCompile(`(?mi)^(?:(?:[a-z0-9._-]+(?:\.[a-z]{2,})?|(?:\d{1,3}\.){3}\d{1,3})(?::\d+)?)/((?:[a-z0-9._-]+/)*)?([a-z0-9._-]+)(:([a-z0-9._-]+))?$`)
```

Reply:
that regex did not workout.... _so_ I borrowed a regex from the `github.com/containers/image` team.... it is just the image reference section but it seems to be a fairly easy drop in replacement for my first regex...

I ran the failing test locally and it is still not happy...

```shell
TMPDIR=/tmp make test-e2e-without-cluster ARCH=amd64
```

So any help on that would be super helpful